<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatValueFieldWidget</class>
 <widget class="QWidget" name="FloatValueFieldWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Float Value Field</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_valueWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="m_nameLabel">
        <property name="text">
         <string>Field:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_typeComboBox">
        <property name="toolTip">
         <string>Type of the value</string>
        </property>
        <item>
         <property name="text">
          <string>val</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inf</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="m_valueSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="m_sepLine">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_serValueWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="m_serFrontLabel">
        <property name="text">
         <string>(0x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_serValueLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="m_serBackLabel">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>31</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
